src/main/java/duke/tasks/Todo.java

```diff
@@ -0,0 +1,45 @@
+package duke.tasks;
+
+/**
+ * Container for a todo task.
+ */
+public class Todo extends Task {
+    /**
+     * Instantiates a Todo object.
+     *
+     * @param description description of the todo task.
+     */
+    public Todo(String description) {
+        super(description);
+    }
+
+    /**
+     * Instantiates a Todo object;
```

Comment:
You know you've been coding too much when you end sentences with a `;` instead of a `.` 😳